Modules/Quotas/src/Quotas.Infrastructure.Database.Postgres/Migrations/20230530160638_TierQuotas.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Quotas.Infrastructure.Database.Postgres.Migrations
+{
+    /// <inheritdoc />
+    public partial class TierQuotas : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "TierQuotaDefinitions",
+                columns: table => new
+                {
+                    Id = table.Column<string>(type: "character(20)", unicode: false, fixedLength: true, maxLength: 20, nullable: false),
+                    MetricKey = table.Column<int>(type: "integer", nullable: false),
+                    Max = table.Column<int>(type: "integer", nullable: false),
+                    Period = table.Column<int>(type: "integer", nullable: false),
+                    TierId = table.Column<string>(type: "character(20)", unicode: false, fixedLength: true, nullable: true)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_TierQuotaDefinitions", x => x.Id);
+                    table.ForeignKey(
+                        name: "FK_TierQuotaDefinitions_Tiers_TierId",
+                        column: x => x.TierId,
+                        principalTable: "Tiers",
+                        principalColumn: "Id");
+                });
+
+            migrationBuilder.CreateTable(
+                name: "TierQuotas",
+                columns: table => new
+                {
+                    Id = table.Column<string>(type: "character(20)", unicode: false, fixedLength: true, maxLength: 20, nullable: false),
+                    _definitionId = table.Column<string>(type: "character(20)", unicode: false, fixedLength: true, maxLength: 20, nullable: true),
```

Comment:
Is it ok to use DataAnnotations?
Not really familiar with a method that adds a custom name through fluent API.

Reply:
No. This would require a reference to EF Core from the domain model. And we want our domain model to be pure.

`HasColumnName` is the method you're looking for. Not that hard to guess. ;)